Fix incorrect instructions for publishing .NET for Apache Spark binaries (#63574)
Azure/azure-content: articles/synapse-analytics/spark/spark-dotnet.md

   ```dotnetcli
   cd mySparkApp
   dotnet publish -c Release -f netcoreapp3.1 -r win-x64
   ```
   
   **On Linux:**

   ```dotnetcli
   cd mySparkApp
   dotnet publish -c Release -f netcoreapp3.1 -r ubuntu.16.04-x64
   ```

> 2. Zip up the contents of the publish folder (say `publish.zip`) created as a result of Step 1 such that all the assemblies are in the first layer of the ZIP file and there is no intermediate folder layer. This means when unzipping `publish.zip` all assemblies should be extracted into the current working directory.

Reviewer: ```suggestion
2. Zip the contents of the publish folder, `publish.zip` for example, that was created as a result of Step 1. All the assemblies should be in the first layer of the ZIP file and there should be no intermediate folder layer. This means when you unzip `publish.zip`, all assemblies are extracted into your current working directory.
```

    **On Windows:**

>     1. Download 7-Zip from [here](https://www.7-zip.org/) and add path to `7z.exe` to the `PATH` environment variable.
>     2. Open command prompt window and cd into the bin directory with all the published binaries and run the following command.
> 
>       ```cmd
>       7z a -tzip <ZIP file name> .
>       ```

Reviewer: ```suggestion
Use an extraction program, like [7-Zip](https://www.7-zip.org/) or [WinZip](https://www.winzip.com/), to extract the file into the bin directory with all the published binaries.
```
